config/default.js

```diff
@@ -0,0 +1,168 @@
+module.exports = {
+	GitterIRCBridges: [
+		{
+			gitter: {
+				token: process.env.NXB_GITTER_TOKEN,
```

Comment:
should refuse to start if you're missing a gitter token. other config options should generate defaults (which would then be saved)